### PR TITLE
Change large SV comparisons to use bar charts instead of timeline

### PR DESCRIPTION
**Pull request by pradh**
Screenshot 2023-01-30 at 5 30 38 PM

### CODE REVIEW

File: server/lib/nl/nl_data_spec_next.py [hunk 1]
```diff
@@ -209,8 +209,12 @@ def populate_simple(uttr: Utterance) -> bool:
 def _populate_simple_cb(state: PopulateState, chart_vars: ChartVars,
                         places: List[Place],
                         chart_origin: ChartOriginType) -> bool:
-  return _add_chart_to_utterance(ChartType.TIMELINE_CHART, state, chart_vars,
-                                 places, chart_origin)
+  if len(chart_vars.svs) <= _MAX_VARS_PER_CHART:
```
Comment (shifucun) [inline server/lib/nl/nl_data_spec_next.py:212]: nit: use if condition to set the chart_type? That makes it clear the only difference is chart type

File: server/lib/nl/nl_data_spec_next.py [hunk 2]
```diff
@@ -638,25 +642,12 @@ def _add_chart_to_utterance(chart_type: ChartType, state: PopulateState,
...
-  start_index = 0
-  while start_index < len(chart_vars.svs):
-    l = min(len(chart_vars.svs) - start_index, _MAX_VARS_PER_CHART)
-    ch = ChartSpec(chart_type=chart_type,
-                   svs=chart_vars.svs[start_index:start_index + l],
-                   places=places,
-                   utterance=state.uttr,
-                   attr=attr)
-    start_index += l
-    state.uttr.chartCandidates.append(ch)
```
Comment (shifucun) [inline server/lib/nl/nl_data_spec_next.py:659]: To confirm, before there could be several charts, now they are all merged into one?

**Review by shifucun — APPROVED** [review]

File: server/lib/nl/nl_data_spec_next.py (hunk 1 above)
Comment (pradh) [reply server/lib/nl/nl_data_spec_next.py:212]: Good idea.

File: server/lib/nl/nl_data_spec_next.py (hunk 2 above)
Comment (pradh) [reply server/lib/nl/nl_data_spec_next.py:659]: Correct.  Yes, before we breakup SVs across multiple time-line charts. Guha wanted them all in one long Bar chart.

**Review by pradh — COMMENTED** [review]
Thanks!

**Pull request merged**